internal/core/keeper/infrastructure/interfaces/mocks/Registry.go

```diff
@@ -0,0 +1,37 @@
+// Code generated by mockery v2.30.1. DO NOT EDIT.
+
+package mocks
+
+import (
+	models "github.com/edgexfoundry/go-mod-core-contracts/v3/models"
+	mock "github.com/stretchr/testify/mock"
+)
+
+// Registry is an autogenerated mock type for the Registry type
+type Registry struct {
```

Comment:
Mocks are do dependencies. This service implements the Registry so it isn't a dependency.